liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/InstallFeatureSupport.java

```diff
@@ -255,9 +287,11 @@ protected InstallFeatureUtil getInstallFeatureUtil(Set<String> pluginListedEsas,
      * @param additionalJsons Collection of Strings for additional jsons for feature install
      * @return instance of InstallFeatureUtil
      */
-    protected InstallFeatureUtil getInstallFeatureUtil(Set<String> pluginListedEsas, List<ProductProperties> propertiesList, String openLibertyVersion, String containerName, List<String> additionalJsons)
+    protected InstallFeatureUtil getInstallFeatureUtil(Set<String> pluginListedEsas, List<ProductProperties> propertiesList, String openLibertyVersion, String containerName, List<String> additionalJsons, Collection<Map<String,String>> keyMap)
             throws PluginExecutionException {
-        createNewInstallFeatureUtil(pluginListedEsas, propertiesList, openLibertyVersion, containerName, additionalJsons);
+    	getLog().info("Verify option: " + features.getVerify());
```

Comment:
Maybe give a little more context for this log message such that it is related to verifying feature signatures. And should it be info or debug?